db/convenience.cc

```diff
@@ -27,6 +27,36 @@ Status DeleteFilesInRange(DB* db, ColumnFamilyHandle* column_family,
       ->DeleteFilesInRange(column_family, begin, end);
 }
 
+Status VerifyChecksum(const std::string& file_path) {
```

Comment:
This is a public function. You should name it as "VerifySstFileChecksum()" or something like that. Otherwise, people don't know what "checksum" it means.